App.js

```diff
@@ -1,20 +1,23 @@
-import { StatusBar } from 'expo-status-bar';
-import { StyleSheet, Text, View } from 'react-native';
+import { createStackNavigator } from "@react-navigation/stack";
+import { NavigationContainer } from "@react-navigation/native";
+import Home from "./screens/Home";
+import FundCustomer from "./screens/FundCustomer";
+import FundEmployer from "./screens/FundEmployer";
+import NewFund from "./screens/NewFund";
+import ViewFund from "./screens/ViewFund";
```

Comment:
This should all come from a index file and be imported together:

`
import { FundCustomer, FundEmployer, NewFund, ViewFund } from './screens'
`